ui/imports/shared/controls/AssetDelegate.qml

```diff
@@ -37,41 +37,35 @@ Item {
             }
         }
     }
+
     StyledText {
         id: assetSymbol
-        text: symbol
         anchors.left: assetInfoImage.right
         anchors.leftMargin: Style.current.smallPadding
         anchors.top: assetInfoImage.top
         anchors.topMargin: 0
         font.pixelSize: 15
+        text: name
     }
+
     StyledText {
         id: assetFullTokenName
-        text: name
         anchors.top: assetSymbol.bottom
         anchors.left: assetInfoImage.right
         anchors.leftMargin: Style.current.smallPadding
-        color: Style.current.secondaryText
         font.pixelSize: 15
+        color: Style.current.secondaryText
+        text: Utils.toLocaleString(enabledNetworkBalance, locale) + " " + symbol.toUpperCase()
```

Comment:
I think I've pointed out already elsewhere, so in short:
`qsTr("L%1 %2").arg(enabledNetworkBalance).arg(symbol)`

The full story: https://hackmd.io/@status-desktop/B1eOaf-nd/%2FjNzZnH8ZRiynrrXJ--XpmQ#Formatting-%E2%80%9Cstuff%E2%80%9D

Reply:
Or, you could simply write:
`enabledNetworkBalance.toLocaleCurrencyString(Qt.locale(), symbol)`

https://doc.qt.io/qt-5/qml-qtqml-number.html#toLocaleCurrencyString-method

Reply:
The whole point is that it's never up to the developer to decide where to put the symbol (before/after), or how to format the number ;)

Reply:
done, looks better now
<img width="693" alt="tk" src="https://user-images.githubusercontent.com/31625338/180750784-7893fff1-0973-4822-b1a3-e1e10e7deac3.png">